vspace/src/main/webapp/WEB-INF/views/staff/modules/slides/slide.html

```diff
@@ -73,7 +74,25 @@
                     $(".open").css('display', 'none');
                     $(".open").css('overflow-y', 'scroll');
                     $(".open").css('height', '400px');
-                } else {
+                } else if(divName.includes("SpaceDivStaff")){
+                	$(e.target).closest('.SpaceDivStaff').addClass("open");
+                    var description = $("div.open p:eq(0)").text();
+                	defaultValue = description;
+            		var editSpaceAlertHTML = document.getElementById('#editSpaceAlert')
+            		var blockId = $(this).closest("div").attr("id");
+            		console.log('block id is'+ blockId)
+            		var selectedSpace = selectedSpaceForSpaceBlock[blockId]
+            		console.log('selected space is '+selectedSpace)
+            		document.getElementById("selectedSpace").innerText = selectedSpace
+                    $("#editSpaceAlert").show();
+                    markDown = new EasyMDE({element: $('#editSpaceBlock')[0]});
+                    markDown.value(description.trim());
+                    //remove card border
+                    $(".open").css('display', 'none');
+                    $(".open").css('overflow-y', 'scroll');
+                    $(".open").css('height', '400px');
+                }
```

Comment:
fix indentations in above block

Reply:
The indentation appears correctly in eclipse i am not sure why it appears like this here.
<img width="645" alt="Screen Shot 2022-02-23 at 12 35 57 PM" src="https://user-images.githubusercontent.com/22446190/155394667-44cac677-bb1a-4ffa-abc2-16349bed10cb.png">